test/unit/notify/send-emails.test.js

```diff
@@ -79,33 +79,9 @@ describe('send emails', () => {
     expect(sendEmail.mock.calls[1]).toEqual([mockContext, mockMessage, defaultEmailAddresses[1], defaultReference])
   })
 
-  test('should throw error when sendEmail rejects', async () => {
```

Comment:
We need to have tests like these to cover the `catch` case in `sendEmails`

You can either:

- raise the Error (preferred) or
- check that `context.log` was called (see `test/integration/local/debt/attach-debt-information.test.js` in `ffc-pay-request-editor`)